src/mudlet.cpp

```diff
@@ -1726,6 +1726,15 @@ void mudlet::closeEvent(QCloseEvent *event)
     qApp->quit();
 }
 
+void mudlet::shutDown()
```

Comment:
I'd probably prefer `forceClose` as the name since it's a close without asking user permission.